flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/streaming/util/TestStreamEnvironment.java

```diff
@@ -129,6 +129,16 @@ private static void randomizeConfiguration(MiniCluster miniCluster, Configuratio
                 boolean enabled =
                         randomize(conf, StateChangelogOptions.ENABLE_STATE_CHANGE_LOG, true, false);
                 if (enabled) {
+                    randomize(
+                            conf,
+                            StateChangelogOptions.PERIODIC_MATERIALIZATION_INTERVAL,
+                            Duration.ofMillis(100),
+                            Duration.ofMillis(500),
+                            Duration.ofSeconds(1),
+                            Duration.ofSeconds(5),
+                            Duration.ofMinutes(10),
+                            Duration.ofSeconds(
+                                    Long.MAX_VALUE / 1000 /* max allowed by Duration.toMillis */));
```

Comment:
This is a good idea.

But do we have ITTests that last more than 10 minutes?

If not I think the last two are equivalent saying that materialization is not played?

Reply:
I think you're right, 10 minutes is probably too much, I'll remove it.